Cargo.toml

```diff
@@ -53,14 +53,15 @@ libc = "0.2.159"
 libc = "0.2.159"
 
 [target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.59"
+version = "0.60"
 features = [
   "Wdk_Foundation",                   # Required for AFD.
   "Wdk_Storage_FileSystem",           # Required for AFD.
   "Wdk_System_IO",                    # Required for AFD.
   "Win32_Foundation",                 # Basic types eg HANDLE
   "Win32_Networking_WinSock",         # winsock2 types/functions
   "Win32_Storage_FileSystem",         # Enables NtCreateFile
+  "Win32_Security",                   # Enables NtCreateFile
```

Comment:
These both say that they enable `NtCreateFile`. Are both needed?

Reply:
Yes, the [definition of `NtCreateFile`](https://github.com/microsoft/windows-rs/blob/99208c88a71ad55dae24565d75e81eb3dba8bfab/crates/libs/sys/src/Windows/Wdk/Storage/FileSystem/mod.rs#L613-L614) has both features:

```rs
#[cfg(all(feature = "Wdk_Foundation", feature = "Win32_Security", feature = "Win32_Storage_FileSystem", feature = "Win32_System_IO"))]
windows_targets::link!("ntdll.dll" "system" fn NtCreateFile(...
```